go/routers/app_multiplexer/tests/test_views.py

```diff
@@ -30,7 +30,11 @@ def test_show_stopped(self):
         router = response.context[0].get('router')
         self.assertEqual(router.name, u"myrouter")
         self.assertContains(response, rtr_helper.get_view_url('start'))
-        self.assertNotContains(response, rtr_helper.get_view_url('stop'))
+        self.assertContains(
+            response,
+            '<button class="btn btn-danger action" data-action="stop" ' +
+            ' data-url="%s" disabled>Deactivate</button>'
+            % rtr_helper.get_view_url('stop'), html=True)
```

Comment:
@jerith @justinvdm I'm not very happy with this either.

The old test said "the URL for the deactivate action shouldn't be in the response."
The new test is supposed to say something like "the deactivate btn (or URL) must be there, but must have the `disabled` attribute." Maybe this test should live with the karma stuff?